tests/zipkin_test.py

```diff
@@ -458,6 +458,24 @@ def test_func(a, b):
     pop_zipkin_attrs_mock.assert_called_once_with()
 
 
+@mock.patch('py_zipkin.zipkin.create_endpoint', wraps=zipkin.create_endpoint)
+def test_zipkin_span_decorator_many(create_endpoint_mock):
```

Comment:
So it seems like you're using `create_endpoint` as a proxy for whether this span tries to log stuff, right? It feels like there should be a better way to do that...